compiler/rustc_mir/src/borrow_check/mod.rs

```diff
@@ -686,7 +686,7 @@ impl<'cx, 'tcx> dataflow::ResultsVisitor<'cx, 'tcx> for MirBorrowckCtxt<'cx, 'tc
                 // "Lift" into the tcx -- once regions are erased, this type should be in the
                 // global arenas; this "lift" operation basically just asserts that is true, but
                 // that is useful later.
-                tcx.lift(&drop_place_ty).unwrap();
+                tcx.lift(drop_place_ty).unwrap();
```

Comment:
Isn't this function call fairly useless now that we only have one global context?

Reply:
Yeah, I don't know what this did but it should be unnecessary. cc @nikomatsakis

Reply:
I agree it can be removed. As the comment states, its only purpose was a kind of assertion, and I think it no longer serves even that purpose.